crypto/bls12381_utils.c

```diff
@@ -519,7 +512,6 @@ int ep2_read_bin_compact(ep2_t a, const byte *bin, const int len) {
     byte compressed = bin[0] >> 7;
     byte y_is_odd = (bin[0] >> 5) & 1;
     if (y_is_odd && (!compressed)) {
-        THROW(ERR_NO_VALID);
```

Comment:
Errors are returned instead of thrown?

Reply:
C doesn't natively support try catch throw, the `THROW` is here is just a macro added by relic library that prints an error message and changes an internal error status. I removed the `THROW`s from these cases because such cases can happen in practise and cause problems when BLS signatures are invalid (that's the case in testnet and mainnet today). It's like panicking in Go for just wrong inputs.